src/spryker/application/skeleton/composer-harness.json.twig

```diff
@@ -68,7 +78,7 @@
     "test-quality": [
       "@generate",
       "@phpstan",
-      "APPLICATION_ENV=development vendor/bin/console code:sniff:style",
+      "APPLICATION_ENV=development vendor/bin/console code:sniff:style src/Inviqa",
```

Comment:
Inviqa namespace would probably not be used by projects, but also Pyz can contain our own stuff too

Reply:
happy though an alternative that does all src/ but Pyz